test/clearingHouse/ClearingHouse.softCircuitBreak.test.ts

```diff
@@ -127,13 +111,10 @@ describe("ClearingHouse softCircuitBreak", () => {
 
             // mint & deposit usdc to davis
             await mintAndDeposit(fixture, davis, 10000)
-
-            // increase blockTimestamp
-            await clearingHouse.setBlockTimestamp(blockTimeStamp + 1)
```

Comment:
what's the reason that we don't need this line?

Reply:
I think it's redundant code statement. In these test case, should do liquidate even the block timestamp is not forward.